www-apps/radicale/radicale-3.2.2.ebuild

```diff
@@ -35,7 +35,7 @@ RDEPEND="
 
 BDEPEND="
 	test? (
-		<dev-python/pytest-8[${PYTHON_USEDEP}]
+		dev-python/pytest[${PYTHON_USEDEP}]
```

Comment:
Let me know if this requires a new revision. I think it does not, but am not sure.

Reply:
It's fine in place.